.github/workflows/pull-request.yml

```diff
@@ -42,11 +42,11 @@ jobs:
 
       - name: Get changed TypeScript files
         id: changed-files
-        uses: tj-actions/changed-files@v45
+        uses: tj-actions/changed-files@v40
```

Comment:
@prayanshchh There are several cases in this file where you reverted the code to a previous version. In this case you downgraded a package that was recently upgraded.